daemons/compute/server/servers/server_default.go

```diff
@@ -308,6 +308,30 @@ Retry:
 	}, nil
 }
 
+func (s *defaultServer) List(ctx context.Context, req *pb.DataMovementListRequest) (*pb.DataMovementListResponse, error) {
```

Comment:
Hmm, where's the list for the NnfDataMovements that are used for the lustre-to-lustre copies?   I think you can let that be in a separate PR, if you prefer.

I would add it to the list that is returned from here.  The RsyncNodeDataMovement and NnfDataMovement resources are named differently, so they won't collide.

You can use the same `opts` list that you've already created here.  That'll work for the NnfDataMovement search as well.

Reply:
I was ignoring this since v2 would deprecate rsync; I can add it then.

Reply:
Just to confirm - I need to get the lists for both DM types and then combine the uids in the result, correct?

Reply:
Merge what you've done so far; we can come back to it in another PR.

Reply:
👍